third_party/chromium/media/filters/chunk_demuxer.cc

```diff
@@ -341,7 +351,7 @@ bool ChunkDemuxerStream::UpdateVideoConfig(const VideoDecoderConfig& config,
   if (!stream_) {
     DCHECK_EQ(state_, UNINITIALIZED);
 #if defined(STARBOARD)
-    stream_ = std::make_unique<SourceBufferStream>(mime_type_, config, media_log);
+    stream_ = std::make_unique<SourceBufferStream>(mime_type_, config, media_log, video_buffer_budget_override_);
```

Comment:
This seems to be very delicate.  Imagining the H5 player is playing two videos, one low res and one high res, using the following steps
1. Set the video budget to 20MB.
2. Create the video element and MediaSource of the low res video.
3. Add SourceBuffer to the low res video.
4. Set the video budget to 100 MB.
5. Create the video element and MediaSource of the high res video.
6. Append data to the low res video, where the config gets parsed, and this function gets called.

Just want to double check that in this case the low res video won't pick 100 MB as its video budget.